.github/workflows/bumpup.yml

```diff
@@ -22,18 +22,13 @@ jobs:
         ruby-version: '3.1'
         bundler-cache: true
 
-    - name: config
-      run: |
-        git config user.name "github-actions[bot]"
-        git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
-
     - name: update sunmodule
       run: git submodule foreach git pull origin master
 
     # ref https://qiita.com/hatsu/items/a7acd14259c3914aad4d
     - name: update gem version
       run: |
-        sed -i "s/VERSION = \".*\"/VERSION = \"${{ github.event.inputs.gem-version }}\"/" lib/jpostcode/version.rb
+        sed -i "s/VERSION = .*/VERSION = ${{ github.event.inputs.gem-version }} /" lib/jpostcode/version.rb
```

Comment:
Probably single quotes will disappear.
https://github.com/kufu/jpostcode-rb/blob/516b5dfd057680e08ce5edb1837d887f80b13904/lib/jpostcode/version.rb#L4

ref. https://bitbucket.org/sedjs/workspace/snippets/7q6r5d

How about using the below regular expression in reference to [semver regular expression](https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string)?
```
\(0\|[1-9][0-9]*\).\(0\|[1-9][0-9]*\)\.\(0\|[1-9][0-9]*\)\.\([0-9]*\)
```

Reply:
This method has the risk of script injections.
ref. https://docs.github.com/ja/actions/security-guides/security-hardening-for-github-actions#understanding-the-risk-of-script-injections